content-cache/src/nginx_manager.py

```diff
@@ -57,6 +64,29 @@
 # This should be tested with integration tests.
 
 
+class NginxLuaSection:  # pylint: disable=R0903
```

Comment:
Sure. I'll do. First I wanted to clarify if this the way we want to go (introducing such a class to handle nginx configuration).

Reply:
Hmm, so what is Lua? I guess that isn't clear to me without knowing the details here

Reply:
Lua is the programming language used to enhance nginx: https://www.lua.org/

Regarding the class, I have update the docstring and used a dataclass to avoid the warning from pylint: 6c070adac1bfc3bf20135d22f57eaf42df6d39c6